moby/moby
# Add OOMScoreAdj to the moby API

**- What I did**
Add OOMScoreAdj to the moby API

**- How I did it**
I modified the executor, convertor and API to accept OOMScoreAdj

**- How to test it**
Using the swarmkit PR
https://github.com/moby/swarmkit/pull/3177

Start moby using make BIND_DIR=. DOCKER_DEBUG=1 DELVE_PORT=127.0.0.1:2345:2345 shell
make install 
./hack/make.sh run

Use curl to query agains the API to verify the changes or you can use the docker CLI from
